src/svgcanvas/event.js

```diff
@@ -1295,7 +1295,7 @@ export const DOMMouseScrollEvent = function (e) {
   if (!e.shiftKey) { return; }
 
   e.preventDefault();
-  const evt = e.originalEvent;
+  const evt = e;
```

Comment:
Why do we need 2 variables ( e and evt) ?

Reply:
We don't need tow variables. I propose this "simple" change only because I don't understand why the original code use the field "originalEvent" which doesn't exists.
If you prefer, I will simplify and use "e" everywhere.

Reply:
Done here : https://github.com/SVG-Edit/svgedit/pull/667/commits/ee8b2c6296164f0f6c1a150d99e7e3a144d73759